src/common/com/intellij/plugins/haxe/model/type/HaxeTypeCompatible.java

```diff
@@ -29,6 +29,19 @@ static public boolean canApplyBinaryOperator(SpecificTypeReference a, SpecificTy
     return true;
   }
 
+  static public boolean isAssignable(SpecificTypeReference a, ResultHolder b) {
```

Comment:
In this file, 'a' and 'b' are used a lot.  However, they don't give me any idea what they are for, or the context in which the functions are to be used.  Is 'a' supposed to be assignable to 'b', or the other way around?  Are they supposed to be interchangeable?  Super-types generally aren't assignable to sub-types, so how do we use this to get proper answers?

Using 'lhs' and 'rhs' for left-hand-side and right-hand-side are common variable names for this idiom and their semantics are generally well understood.  Better may be 'assignee' and 'assignableTo'.  Either way, this model is confusing to me and it needs some documentation, along with example usage in a JavaDoc comment.